include/aspect/particle/property/integrated_strain.h

```diff
@@ -96,6 +95,14 @@ namespace aspect
           need_update () const;
 
           /**
+           * Return, which data has to be provided to update the property.
+           * The integrated strains needs the gradients of the velocity.
+           */
+          virtual
+          std::vector<UpdateFlags>
+          get_needed_update_flags () const;
```

Comment:
`UpdateFlags` is a binary mask, i.e., you can say `update_values|update_gradients` and it is still an `UpdateFlags`. Why return a vector, instead of just an `UpdateFlags` variable?